src/app/routes/_question.router.js

```diff
@@ -163,6 +163,20 @@ export default (app, router, io, admin) => {
         return;
       }
 
+      const youtubeData = await youtubeHelper.getLivestreamInfo();
+      // check if broadcast is running and try to set the time this question has been answered
+      if (youtubeData.items.length > 0) {
+        const stream = youtubeData.items[0];
+        const streamStarted = moment(stream.snippet.publishedAt);
+        const now = moment();
+        const duration = moment.duration(now.diff(streamStarted));
+
+        console.log('Stream Started', streamStarted.toString());
+        console.log('Now', now.toString());
+        console.log('Diff in s', Math.round(duration.asSeconds()));
```

Comment:
Yes, because I can't really test this in dev currently. We'd need to do a prod test with that, I guess.